xcode/Subconscious/SubconsciousTests/Tests_NoosphereService.swift

```diff
@@ -21,7 +21,45 @@ final class Tests_NoosphereService: XCTestCase {
         )
         return url
     }
+    
+    func testSphereTraversal() throws {
+        throw XCTSkip("Sphere Traversal relies on running a gateway, this test will not pass currently.")
```

Comment:
Test is skipped for now, but this demonstrates the intended usage for `traverse`